src/pages/components/table-of-contents.mdx

```diff
@@ -42,14 +51,26 @@ The area above the Table of contents can be utilized in many ways—showing navi
 <Row>
 <Column colMd={8} colLg={8}>
 
-![Image of Table of contents component](../../images/component/table-of-contents/table-of-content-with-heading-content.jpg)
+![Image of With heading content variation](../../images/component/table-of-contents/table-of-content-with-heading-content.jpg)
 
 </Column>
 </Row>
 
 <Title>Use case</Title>
 
-![Image of Table of contents component](../../images/component/table-of-contents/table-of-content-with-heading-content-use-case.jpg)
+![Image of With heading content variation usecase](../../images/component/table-of-contents/table-of-content-with-heading-content-use-case.jpg)
+
+
+### Horizontal
+The Table of Contents – Horizontal variation renders as horizontal navigation that includes anchor links to various locations on the page. It sticks to the top of the page when scrolling and pushes the masthead navigation out of the browser window.
+
+![Image of Horizontal variation](../../images/component/table-of-contents/table-of-contents-horizontal.jpg)
+
```

Comment:
Do we have dos and dont's for this component? Are they the same as the above? If they are the same (no reason to duplicate) could we move the above dos and don'ts higher up in the page?

Reply:
I thought I added a when to use on here, but not seeing it for some reason. 

I considered adding something about using the horizontal version with Section level components.